src/quo2/components/messages_gap.cljs

```diff
@@ -0,0 +1,155 @@
+(ns quo2.components.messages-gap
+  (:require
+   [oops.core :refer [oget]]
+   [quo.react-native :as rn]
+   [quo.theme :as theme]
+   [quo2.components.text :as text]
+   [quo2.foundations.colors :as colors]
+   [reagent.core :as reagent]
+   [status-im.i18n.i18n :as i18n]
+   [status-im.ui.components.react :refer [pressable-class]]
+   [status-im.utils.handlers :refer [>evt]]))
+
+;;; helpers
+(def themes
+  {:light {:icon       colors/neutral-40
+           :time       colors/neutral-50
+           :background colors/neutral-5}
+   :dark  {:icon       colors/neutral-60
+           :time       colors/neutral-40
+           :background colors/neutral-95}})
+
+(defn get-color [key]
+  (get-in themes [(theme/get-theme) key]))
+
+(def info-img (js/require "../resources/images/icons/message-gap-info12.png"))
```

Comment:
this looks like an icon, it required , so you can just use `icon` component from quo2